app/views/application/_navigation.html.haml

```diff
@@ -14,7 +14,7 @@
       - else
         .div.custom
           .nav-item
-            = link_to new_post_path, class: 'nav-link', data: { 'toggle': 'tooltip', 'original-title': 'Create post' } do
+            = link_to new_post_path, class: 'nav-link', data: { 'toggle': 'tooltip', 'original-title': 'Create post', id: 'link-to-new-post' } do
```

Comment:
keep the format of the `id` as `'id'`. It'll be a symbol but this way it keep the same format as the others